docs/src/app/components/pages/components/IconButton/ExampleComplex.jsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import FontIcon from 'material-ui/lib/font-icon';
 import IconButton from 'material-ui/lib/icon-button';
+import ActionHome from 'material-ui/lib/svg-icons/action/schedule';
```

Comment:
Although you could argue that while possible, this method is redundant given that `IconButton` supports the `iconClassName` prop, so we could just leave it out.